components/session-timeout-warning/Readme.md

```diff
@@ -0,0 +1,43 @@
+# Session Timeout Warning Behaviour
+
+## What this does
+Makes the content related to the session timeout warning behaviour customisable. This includes the content in the session timeout warning dialog and on the exit page after the user clicks to exit the form from the session timeout warning dialog.
+
+The component should be set in the project's hof.settings.json
+```
+ "behaviours": [
+    "hof/components/session-timeout-warning"
+  ]
+```
+
+By default,the default content from the hof framework. If you require customised content at a project level, the following variables must be set to true in hof.settings.json
```

Comment:
```suggestion
By default, the framework uses the standard content provided by HOF. If you wish to override this with custom content at the project level, you must set the following variables to `true` in `hof.settings.json`:
```

Reply:
I have updated this with your content suggestions (can now be found in main readme)

Reply:
I have updated this with your content suggestions (can now be found in main readme)

Reply:
nice one